src/assets/css/_sass/elements/_buttons.scss

```diff
@@ -5,3 +5,23 @@
   Don't forget hover, active states
 
 */
+
+.button {
+  padding: .575rem .75rem;
```

Comment:
use rem's with caution - there is no Root element font size set, so these use the default of the browser. 16px usually, but can be different. Here you have some infinite decimal 9.... px and a 12px padding